package.json

```
@@ -9,7 +9,8 @@
     ".": {
       "require": "./dist/index.js",
       "import": "./dist/index.mjs"
-    }
+    },
+    "./": "./"
```

this syntax is now deprecated (see [docs](https://nodejs.org/api/packages.html#packages_subpath_folder_mappings)) and will print an error with node 16+.

If it's just the `package.json` that will need to be accessed then we can provide for it like this:
```suggestion
    "./package.json": "./package.json"
```

---

Thanks for the heads up! Unfortunately it is not the `package.json` but `Package subpath './dist/index.mjs'`. Will change to subpath patterns with wildcard